app/src/main/java/com/github/sdp/ratemyepfl/database/RestaurantRepository.kt

```diff
@@ -33,4 +27,29 @@ class RestaurantRepository @Inject constructor() : RestaurantRepositoryInterface
     }
 
     override suspend fun getRestaurantById(id: String): Restaurant? = toItem(getById(id))
+
+    override fun incrementOccupancy(id: String) {
```

Comment:
Great job for the transactions ! Your code really helped me understand how it works. I used it for overall grade in the review part of the app :).

Reply:
Glad it helped ! :)